ovirt_imageio/client/_upload.py

```diff
@@ -129,7 +134,7 @@ def _prepare(args):
     name = os.path.splitext(os.path.basename(img_info["filename"]))[0]
```

Comment:
If user specified a namme, we don't need to extract the name from the file name, so better do:

    if args.name is None:
        name = os.path.splitext(os.path.basename(img_info["filename"]))[0]
    else:
        name = args.name